lib/auth/webauthn/login_test.go

```diff
@@ -445,9 +445,7 @@ func TestPasswordlessFlow_BeginAndFinish(t *testing.T) {
 					AllowReuse: mfav1.ChallengeAllowReuse_CHALLENGE_ALLOW_REUSE_NO,
 				},
 			}
-			if diff := cmp.Diff(wantSD, sd); diff != "" {
-				t.Fatalf("SessionData mismatch (-want +got):\n%s", diff)
-			}
+			require.Empty(t, cmp.Diff(wantSD, sd), "SessionData mismatch")
```

Comment:
nit: Revert this file.

There's no need to shift blame to you for a noop change. (Also testify's error message is way worse.)